packages/jest-config/src/ValidConfig.ts

```diff
@@ -7,14 +7,14 @@
 
 import {Config} from '@jest/types';
 import {replacePathSepForRegex} from 'jest-regex-util';
-// @ts-ignore: Not migrated to TS
 import {multipleValidOptions} from 'jest-validate';
 import {NODE_MODULES} from './constants';
 
 const NODE_MODULES_REGEXP = replacePathSepForRegex(NODE_MODULES);
 
 const initialOptions: Config.InitialOptions = {
   automock: false,
+  // @ts-ignore TODO: type this properly
```

Comment:
@jeysal do you know how to type this? Essentially, the return-type should be one a union of the types of all arguments

Reply:
Not sure I understand, wouldn't that typing be a lie? `multipleValidOptions` doesn't return one of the argument (in that case the return type would be the union of them), but an array of all of them.